sda-commons-server-cloudevents/src/main/java/org/sdase/commons/server/cloudevents/CloudEventV1.java

```diff
@@ -0,0 +1,181 @@
+          + "for more information.")
+  @NotEmpty
+  private String type;
+
+  @JsonPropertyDescription(
+      "This describes the subject of the event in the context of the event "
+          + "producer (identified by `source`). In publish-subscribe scenarios, a subscriber will "
+          + "typically subscribe to events emitted by a source, but the `source` identifier alone might "
+          + "not be sufficient as a qualifier for any specific event if the `source` context has "
+          + "internal sub-structure.\n"
+          + "\n"
+          + "Identifying the subject of the event in context metadata (opposed to only in the data "
+          + "payload) is particularly helpful in generic subscription filtering scenarios where "
+          + "middleware is unable to interpret the data content. In the above example, the subscriber "
+          + "might only be interested in blobs with names ending with '.jpg' or '.jpeg' and the "
+          + "subject attribute allows for constructing a simple and efficient string-suffix filter "
+          + "for that subset of events.")
+  private String subject;
+
+  @JsonPropertyDescription(
+      "Content type of data value. This attribute enables data to carry any "
+          + "type of content, whereby format and encoding might differ from that of the chosen event "
+          + "format. For example, an event rendered using the JSON envelope format might carry an XML "
+          + "payload in data, and the consumer is informed by this attribute being set to "
+          + "\"application/xml\". The rules for how data content is rendered for different "
+          + "`datacontenttype` values are defined in the event format specifications; for example, the "
+          + "JSON event format defines the relationship in section 3.1.\n"
+          + "\n"
+          + "For some binary mode protocol bindings, this field is directly mapped to the respective "
+          + "protocol's content-type metadata property. Normative rules for the binary mode and the "
+          + "content-type metadata mapping can be found in the respective protocol\n"
+          + "\n"
+          + "In some event formats the `datacontenttype` attribute MAY be omitted. For example, if a "
+          + "JSON format event has no `datacontenttype` attribute, then it is implied that the data is "
+          + "a JSON value conforming to the \"application/json\" media type. In other words: a "
+          + "JSON-format event with no `datacontenttype` is exactly equivalent to one with "
+          + "`datacontenttype=\"application/json\"`.\n"
+          + "\n"
+          + "When translating an event message with no `datacontenttype` attribute to a different format "
+          + "or protocol binding, the target `datacontenttype` SHOULD be set explicitly to the implied "
+          + "`datacontenttype` of the source.")
+  private String datacontenttype;
```

Comment:
At SDA we always use application/json. Maybe a good idea would be to enforce this by removing the "set" method and hard-code the value here (bind it with the setData method in case that the value should be null).

Reply:
I set the default value to `application/json` but kept the setters. If really someone wants a different value, it's still possible.

Reply:
The only thing, I could imagine is something like `application/my.custom.spec+json`. I appreciate that this is still possible 👍